src/components/sidebar/helpers/generate-top-level-sidebar-nav-data.ts

```diff
@@ -1,11 +1,10 @@
-import { ProductName } from 'types/products'
 import { generateTopLevelSubNavItems } from 'lib/generate-top-level-sub-nav-items'
 
 /**
  * Generates the top-level website nav data for rendering in `Sidebar` as the
  * top-level of the mobile navigation experience.
  */
-export const generateTopLevelSidebarNavData = (productName: ProductName) => {
+export const generateTopLevelSidebarNavData = (productName: string) => {
```

Comment:
I figured it wasn't actually important that this be a product name specifically.

Reply:
💭 Longer term I think it'll be nice to move away from product-specific terminology to something like "topic", but something like that probably needs some time to cook in everyone's brain ovens. 😁 🧠

Reply:
Agreed! Let's stew on that a bit 🥣